src/views/Header.js

```diff
@@ -80,6 +81,17 @@ class Header extends React.PureComponent<void, HeaderProps, HeaderState> {
     ).options.headerTruncatedBackTitle;
   }
 
+  _getBackButtonTitleStyle(scene: NavigationScene): ?Style {
+    const lastScene = this._getLastScene(scene);
+    const { headerBackTitleStyle } = this.props.getScreenDetails(scene).options;
+    if (!lastScene) {
+      return headerBackTitleStyle;
+    }
+    const { headerTitleStyle } = this.props.getScreenDetails(lastScene).options;
+
+    return [headerTitleStyle, headerBackTitleStyle];
```

Comment:
> You always want the font family and color to be consistent with the current screen.

IMO it's not for nothing that the titles moves to the position of the back button; it's supposed to match up with the title of the last screen, and doing it like this will prevent the need of supplying anything.

What do you think about an option which defines whether or not the style of the previous scene should be used?